### DISCUSSION TITLE
Service Release 2

### DISCUSSION THREAD

**Discussion by yairp03** [dotnet/maui#8229] [Q&A] [answered]
Will service release 2 be released today? And if so, will it be available immediately on Visual Studio 2022 preview?

**Comment by hartez** [comment, marked as answer]
It will not be released today. Today is the last day for developers to submit pull requests for SR2; the final version of SR2 should be tagged next week and released some time after that.

**Reply by danielftz** [reply]
Any update on when SR2 is coming out?

**Reply by hartez** [reply]
Currently targeting next week.

**Reply by yairp03** [reply]
@danielftz It seems like there is a release once a month, so check the last releases and you can estimate the date of the next one
